Part2_Mapplots/ex2_0_intro_scatter.ipynb

```diff
@@ -130,7 +130,7 @@
    "source": [
     "We created our first map plot!\n",
     "\n",
-    "Unfortunately, writing `f, ax = plt.subplots(subplot_kw=dict(projection=projection))` is a bit cumbersome. I recommend you just copy-paste it.\n",
+    "Unfortunately, writing `f, ax = plt.subplots(subplot_kw=dict(projection=projection))` is a bit cumbersome. I recommend you to just copy-paste it.\n",
```

Comment:
I would actually remove this sentence. The users don't need the remark to copy-paste it and also the copy-paste doesn't work as one also needs to add 1,2, into plt.subplots(). So the remark can be even misleading.

Reply:
Ok, I'll remove it and mention the `plt.axes(projection=...)` syntax.